pkg/prometheus/promcfg_test.go

```diff
@@ -8837,3 +8873,95 @@ scrape_configs:
 
 	}
 }
+
+func TestTracingConfig(t *testing.T) {
+	boolFalse := false
+	samplingFraction := "2"
+	clientType := "grpc"
+	compression := "gzip"
```

Comment:
Instead of declaring all those variables here, you can use `pointer.Bool(false)`, `pointer.Int(2)`, `pointer.String("grpc")` where needed.

The failing test is probably related to a copy-paste mistake when declaring the variables above 😅

Reply:
Oh!
That's cool.
TIL